mobile/src/bundles/common/styles/flex.ts

```diff
@@ -0,0 +1,71 @@
+import { type FlexAlignType, type FlexStyle, StyleSheet } from 'react-native';
+
+const AlignmentValues = {
+    CENTER: 'center',
+    FLEX_START: 'flex-start',
+    FLEX_END: 'flex-end',
+    STRETCH: 'stretch',
+    SPACE_BETWEEN: 'space-between',
+    SPACE_AROUND: 'space-around',
+    SPACE_EVENLY: 'space-evenly,',
+} as const;
+
+const flex = StyleSheet.create({
```

Comment:
It should be just a simple object to avoid nested StyleSheet.create 
```suggestion
const FLEX_STYLES = {
```